components/mail/mutt/mutt.p5m

```diff
@@ -19,18 +19,9 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
-# Copyright 2018, Michal Nowak
+# Copyright 2019, Michal Nowak
 #
 
-<transform file path=usr.*/man/.+/flea\.1$ -> default mangler.man.stability uncommitted>
-<transform file path=usr.*/man/.+/mutt\.1$ -> default mangler.man.stability uncommitted>
-<transform file path=usr.*/man/.+/muttbug\.1$ -> default mangler.man.stability uncommitted>
-<transform file path=usr.*/man/.+/pgpewrap\.1$ -> default mangler.man.stability volatile>
-<transform file path=usr.*/man/.+/pgpring\.1$ -> default mangler.man.stability volatile>
-<transform file path=usr.*/man/.+/smime_keys\.1$ -> default mangler.man.stability volatile>
-<transform file path=usr.*/man/.+/mbox\.5$ -> default mangler.man.stability committed>
-<transform file path=usr.*/man/.+/mmdf\.5$ -> default mangler.man.stability committed>
-<transform file path=usr.*/man/.+/muttrc\.5$ -> default mangler.man.stability volatile>
```

Comment:
Why is this no longer needed?

Reply:
I think, because we usually can't guarantee the stability of interface provided by  upstream project. What if it changes at some point? Will we patch it? I doubt we even notice it. So, I think stability of third-party-provided interfaces is mostly a legacy artefact. Of course we can preserve this, but I don't think this makes much sense.

Reply:
Right, we cannot guarantee the stability, so it is set to uncommitted or volatile.  The only committed stability is for something (mbox and mmdf) that is really stable.  The documented stability is very helpful in many cases and sets user's (developer's) expectations properly.  Please keep it, if possible.  Thanks.

Reply:
> I think, because we usually can't guarantee the stability of interface provided by upstream project. What if it changes at some point? Will we patch it? I doubt we even notice it. So, I think stability of third-party-provided interfaces is mostly a legacy artefact. Of course we can preserve this, but I don't think this makes much sense.

Yes, that's exactly my reasoning behind this change.

> Right, we cannot guarantee the stability, so it is set to uncommitted or volatile. The only committed stability is for something (mbox and mmdf) that is really stable. The documented stability is very helpful in many cases and sets user's (developer's) expectations properly. Please keep it, if possible. Thanks.

For that I need to see upstream's explicit commitment to stability of those component's interfaces, otherwise we are misleading developers in believing this information is to be trusted. Even in case of interfaces being stable in every practical regard, without upstream guaranteeing it, they can change without notice. Personally, I remove those "guarantees" wherever I see them.